Tests/test_PDB.py

```diff
@@ -1576,6 +1576,20 @@ def test_pdb_to_xyzr(self):
         assert len(msms_radii) == len(biopy_radii)
         self.assertSequenceEqual(msms_radii, biopy_radii)
 
+class FragmentMapperTests(unittest.TestCase):
+    """ Tests for FragmentMapper module.""""
```

Comment:
This is invalid syntax - there should be three closing double-quote characters.

Also please remove the space before the word 'Tests'.